src/presets/index.js

```diff
@@ -0,0 +1,55 @@
+const SPAIN = require("./population/SpainPreset");
+const ITALY = require("./population/ItalyPreset");
+const GERMANY = require("./population/GermanyPreset");
+
+const CONTROL = require("./simulation/ControlPreset");
+const GOOD_HYGIENE = require("./simulation/GoodHygienePreset");
+const RELAXED_QUARANTINE = require("./simulation/RelaxedQuarantinePreset");
+const STRICT_QUARANTINE = require("./simulation/RelaxedQuarantinePreset");
+const CARRIERS_QUARANTINE = require("./simulation/RelaxedQuarantinePreset");
+
+
+/**
+ * simulation preset structure
+ * {
+ *  populationSize: *number*, in N and > 9, such that sqrt(populationSize) in N,
+ *  hospitalEffectiveness: *number*, in [0, 1]
+ *  hygieneDisregard: *number*, in [0, 1] | default 1
+ *  hospitalCapacity: *number* , in [0, 1]
+ *  incubationPeriod: *number*, in [0, 50] and in N | default 6
+ *  infectionPeriod: *number*, in [2, 100] and in N | default 41
+ *  spreadRadius: *number*, in [2, sqrt(populationSize)] | default 1
+ *  testRate: *number*, in [0, 1] | default 0
+ *  testCooldown: *number* in [1, 100] and in N | default 7
+ *  quarantineType: *number* in [NONE, FIXED_PERCENTAGE, CONFIRMED_CARRIERS_ONLY],
+ *  quarantineEffectiveness: *number*, in [0, 1]
+ *  quarantinePercentage: (optional) *number*, in [0, 1]
+ *  quarantineDelay: *number*, in [0, 50] and in N,
+ *  quarantinePeriod: *number*, in [0, 50] and in N,
+ *  populationPreset, *number*, in [0, 1, 2]
+ * }
+ *
+ */
+const SimulationPresets = Object.freeze({
+    CONTROL,
+    GOOD_HYGIENE,
+    RELAXED_QUARANTINE,
+    STRICT_QUARANTINE,
+    CARRIERS_QUARANTINE,
+});
+
+/**
+ * population preset structure
+ * {
+ *  size: *number*,
+ *  susceptibilityDistibution: {percentage*number*:susceptibility*number*},
+ *  label: *string*,
+ * }
+ *
+ */
+const PopulationPresets = [SPAIN, ITALY, GERMANY];
```

Comment:
It might be better to have an object with {populationID: presetObject} instead of an array, so that we can reference them by a meaningful string instead of a meaningless number. Can you do that refactor?

Reply:
You will also need to update the init() function to deal with a string instead of number

Reply:
Yes, i think it is done! I did not change init the access through index and through a key on an object have the same syntax. Is this what you meant by dealing with a string?

Reply:
Yes, since populationPresets is now an object with "SPAIN": <something>, "ITALY": <something>, etc, you need to access it via the countries string, instead of a number, it's as simple as updating the config.js to read string from env, instead of numbers on the POPULATION_PRESET option, the init should work the same way, as it uses that variable directly, but you can check that as well